website/package.json

```diff
@@ -21,39 +21,40 @@
   "author": "Patronage.org",
   "license": "private",
   "dependencies": {
-    "@popperjs/core": "^2.9.3",
-    "bootstrap": "^5.1.0",
+    "@popperjs/core": "^2.10.2",
+    "bootstrap": "^5.1.3",
     "classnames": "^2.3.1",
-    "dayjs": "^1.10.6",
+    "dayjs": "^1.10.7",
     "isomorphic-unfetch": "^3.1.0",
-    "next": "^11.1.0",
-    "next-seo": "^4.26.0",
+    "next": "^12.0.4",
+    "next-seo": "^4.28.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-icons": "^4.2.0",
-    "sass": "^1.38.1",
-    "slugify": "^1.6.0",
+    "react-icons": "^4.3.1",
+    "sass": "^1.43.4",
+    "slugify": "^1.6.2",
     "widont": "^0.3.3"
   },
   "devDependencies": {
-    "@next/bundle-analyzer": "^11.1.0",
+    "@next/bundle-analyzer": "^12.0.4",
     "babel-eslint": "^10.1.0",
     "cross-env": "^7.0.3",
-    "eslint": "^7.32.0",
-    "eslint-config-next": "^11.1.0",
+    "eslint": "<8.0.0",
+    "eslint-config-next": "^12.0.4",
     "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-import": "^2.24.2",
-    "eslint-plugin-react": "^7.24.0",
-    "eslint-plugin-react-hooks": "^4.2.0",
+    "eslint-plugin-import": "^2.25.3",
+    "eslint-plugin-react": "^7.27.1",
+    "eslint-plugin-react-hooks": "^4.3.0",
     "husky": "^7.0.4",
     "lint-staged": "^12.0.3",
-    "prettier": "^2.3.2",
-    "stylelint": "^13.13.1",
+    "prettier": "^2.4.1",
+    "stylelint": "^14.1.0",
     "stylelint-config-idiomatic-order": "^8.1.0",
-    "stylelint-config-prettier": "^8.0.2",
-    "stylelint-config-standard": "^22.0.0",
-    "stylelint-prettier": "^1.2.0",
-    "stylelint-scss": "^3.20.1"
+    "stylelint-config-prettier": "^9.0.3",
+    "stylelint-config-standard": "^24.0.0",
+    "stylelint-config-standard-scss": "^2.0.1",
```

Comment:
just updated to 3.0.0 three days ago which uses `stylelint-config-standard@24.0.0` and doesn't have any breaking changes so seems like a good update

Reply:
does this mean we can just use a single dep in here (and remove `stylelint-config-standard`)?

Reply:
yep, its in `stylelint-config-standard-scss`'s dependencies